src/Features/Core/Portable/UpgradeProject/AbstractUpgradeProjectCodeFixProvider.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using static Microsoft.CodeAnalysis.CodeActions.CodeAction;
+
+namespace Microsoft.CodeAnalysis.UpgradeProject
+{
+    internal abstract partial class AbstractUpgradeProjectCodeFixProvider : CodeFixProvider
+    {
+        public override Task RegisterCodeFixesAsync(CodeFixContext context)
+        {
+            var diagnostics = context.Diagnostics;
+
+            context.RegisterFixes(GetUpgradeProjectCodeActionsAsync(context), diagnostics);
+            return Task.CompletedTask;
+        }
+
+        protected ImmutableArray<CodeAction> GetUpgradeProjectCodeActionsAsync(CodeFixContext context)
+        {
+            var project = context.Document.Project;
+            var solution = project.Solution;
+            var newVersions = SuggestedVersions(context.Diagnostics);
+            var result = new List<CodeAction>();
+
+            foreach (var newVersion in newVersions)
+            {
+                var fixOneProjectTitle = string.Format(UpgradeThisProjectResource, newVersion);
+
+                CodeAction fixOneProject = new ParseOptionsChangeAction(fixOneProjectTitle,
+                    ct => Task.FromResult(UpgradeProject(solution, project.Id, newVersion)));
+
+                result.Add(fixOneProject);
+
+                if (solution.Projects.Where(CouldBeUpgradedToo).Count() > 1)
```

Comment:
No. I'm not trying to check `Count() > 0` (which is always try at this point in the code), but `Count() > 1`.